addons/mail/static/src/models/call_view.js

```diff
@@ -137,19 +137,12 @@ registerModel({
                 return clear();
             }
             const tileCards = [];
-            for (const rtcSession of this.channel.rtcSessions) {
-                if (this.filterVideoGrid && !rtcSession.videoStream) {
+            for (const channelMember of this.channel.channel.callParticipants) {
+                if (this.filterVideoGrid && !channelMember.isStreaming) {
                     continue;
                 }
                 tileCards.push({
-                    rtcSession: replace(rtcSession),
-                    channel: replace(this.channel),
-                });
-            }
-            for (const member of this.channel.invitedMembers) {
```

Comment:
I suppose it's ok to change, but according to diff it means previously cards were created for invited members even if `filterVideoGrid` was true?